client/libs/material/src/lib/sermon-table/sermon-table-datasource.ts

```diff
@@ -0,0 +1,68 @@
+import { DataSource } from '@angular/cdk/collections';
+import { MatPaginator } from '@angular/material/paginator';
+import { MatSort } from '@angular/material/sort';
+import { map } from 'rxjs/operators';
+import { Observable, of, merge } from 'rxjs';
+
+// tslint:disable-next-line: nx-enforce-module-boundaries
```

Comment:
Try to come up with a solution for the circular dependencies here.

Maybe try editing the paths in `tsconfig`?